policy/modules/contrib/kpatch.if

```diff
@@ -39,6 +39,26 @@ interface(`kpatch_nnp_domtrans',`
 
 ')
 
+########################################
+## <summary>
+##	Read kpatch lib files
+## </summary>
+## <param name="domain">
+## <summary>
+##	Domain allowed access
+## </summary>
+## </param>
+#
+interface(`kpatch_read_lib_files',`
+	gen_require(`
+		type kpatch_var_lib_t;
+	')
+
+	files_search_var_lib($1)
+	allow $1 kpatch_var_lib_t:dir list_dir_perms;
```

Comment:
Maybe you could use `list_dirs_pattern($1, kpatch_var_lib_t, kpatch_var_lib_t)` instead, to be consistent with the line below? The resulting permission set should be the same.

Reply:
Surely I can. I did not do so because of the duplicate definition inside list_dirs_pattern.